code/parachain/frame/staking-rewards/src/test/test_reward_accumulation_hook.rs

```diff
@@ -148,6 +139,7 @@ fn test_accumulate_rewards_pool_empty_refill() {
 		// cap will be hit after 5_000 seconds (834 blocks)
 		const A_A_REWARD_RATE: u128 = A::units(2) / 100_000;
 		const A_A_MAX_REWARDS: u128 = A::units(1) / 10;
+		const A_A_AMOUNT_TO_ADD: u128 = A_A_REWARD_RATE * block_seconds(4);
```

Comment:
I would rename this to something slightly more descriptive - `A_A_AMOUNT_TO_ADD_TO_REWARDS_POT`; I was confused as to what 'amount to add' meant until I saw it used